airflow/www/fab_security/sqla/models.py

```diff
@@ -53,41 +43,11 @@
         Identity = None
 
 
-def get_sequence_or_identity(sequence_name: str) -> Union[Sequence, 'Identity']:
```

Comment:
`Sequence` is not used in migrations. Some DB(MariaDB) try to add the sequence at this migration https://github.com/apache/airflow/blob/2.3.1/airflow/migrations/versions/0073_2_0_0_prefix_dag_permissions.py but shouldn't. With the sequence in place, creating table directly from ORM, removes autoincrement default value in postgres. For both ORM and migration to be the same, this should be removed. 
Some discussions: https://github.com/apache/airflow/issues/24247#issuecomment-1147783514